Resources/Prototypes/_Goobstation/GameRules/secretplusAlternate.yml

```
@@ -0,0 +1,241 @@
+
+# Tables
+
+- type: entity
+  parent: BaseGameRule
+  id: SubGamemodesRuleClassic # Currently kinda scuffed w/ only 2 side antags, when there's 3-6 it should work better- try and make it select 2 side antags on average when there's 3+
+  components:
+  - type: SubGamemodes
+    rules: # CurrentlY 37.5% chance of thief alone, 37.5% both, 12.5% Devil alone, 12.5% 0 antags- gonna be 10x harder to calculate later lol
+    - id: Thief
+      prob: 0.75 # Decrease to 0.65-0.4 when there's more antags
+    - id: Devil 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)